app/workers/education_form/create_daily_fiscal_year_to_date_report.rb

```diff
@@ -234,7 +234,7 @@ def generate_csv
     def perform
       Sentry::TagRainbows.tag
       filename = generate_csv
-      return unless FeatureFlipper.send_email?
+      return unless FeatureFlipper.send_email && !Rails.env.development?
```

Comment:
the rails env for staging/dev is `production`, still need to use the hostname method

Reply:
ok, made it simpler now to use hostname check to not include `dev-api`